scripts/run-integration-tests-locally.sh

```diff
@@ -67,7 +67,7 @@ function fund_node {
   fi
 
   echo "- Funding 1 ETH and 1 HOPR to $ETH"
-  $hardhat faucet --config packages/ethereum/hardhat.config.ts --address "$ETH" --network localhost --ishopraddress true
+  $hardhat faucet --config packages/ethereum/hardhat.config.ts --address "$ETH" --network hardhat --ishopraddress true
```

Comment:
no need to specify network, will pick hardhat by default

Reply:
On things like this, I'd prefer to be specific as it cuts down mental load when debugging - the fact hardhat is default isn't obvious until you read the hardhat docs.